io/CMakeLists.txt

```
@@ -415,7 +415,7 @@ if(PCAP_FOUND)
   target_link_libraries("${LIB_NAME}" ${PCAP_LIBRARIES})
 endif()
 
-set(EXT_DEPS boost eigen3)
+set(EXT_DEPS eigen3)
```

```suggestion
set(EXT_DEPS eigen3) # Although this depends on boost, that cannot be specified here because there is no boost.pc
```
Just an idea :slightly_smiling_face:

---

Well all of PCL depends on boost and eigen3 so this should rather go into common, or?

---

You mean just the comment or the whole `set(EXT_DEPS eigen3)`? Because I just noticed that common does not list any external dependencies, although it probably should list eigen3?
To be fair, the comment that there is no boost.pc is just a small detail - it could theoretically be added in all PCL modules (overkill IMO), it just seemed to make sense to add it here because we mistakenly added boost here

---

I meant all of it and I agree that common should list eigen3.
Anyway, feel free add your comment so we can get this merged :).